### ISSUE TITLE
understanding the use case

### ISSUE THREAD

**Issue by ninrod** [willghatch/emacs-repeatable-motion#4]
Hi @willghatch, I don't think I understand the use case correctly. 

say I issued `9j`. Can I repeat this with you plugin?

**Comment by willghatch** [comment]
Yes, although I don't automatically bind anything.  Perhaps I should provide a function that binds everything for evil users, but every time I use a plugin that automatically binds things the first thing I do is figure out where things are bound and undo their work in binding things so I can set up my own bindings.

So there are two main use cases of the plugin.  One is repeating something like `9j`, and one is to be able to bind less common motions to longer key sequences but be able to repeat them with a single key press.  So you can, say, bind `repeatable-motion-forward` to `;`, then you make `h` and `l` prefixes for left and right movements -- maybe `hp` moves backwards a paragraph and `lp` moves forward a paragraph, `hs` and `ls` for sentence, etc, and probably including a whole bunch of motion commands that aren't in vim (move to next argument/form, forward little word, forward to end of s-expression, etc).  Then you have to use 2 (or maybe more) keys to do the motion once, but to spam repeating it you just press `;`.  This gives you the ability to bind a lot more movement commands, and frees up a lot of single keys for other common commands you might use, while still being able to repeat the movement with just one key.

The latter is what I really use it for -- I never use numeric arguments unless I'm repeating a keyboard macro and I conciously stop beforehand to see how many times I need to run it.  I think the only movements I have bound to single letters are `j`, `k`, `w`, `b`, `/`, and `?`, since I use them the most.  This has freed up my layout a lot to fit other things in, and it's been easier for me to remember different movement commands because they all have the same prefix and are mostly more mnemonic with their new sub-keys than with their original keys.

**Comment by ninrod** [comment]
that's awesome. but I did not understand how to setup the plugin to achieve the desired effect. 

Let's say I want that all motions I issue to evil be reapeatable. Like, `C-f`, `9j`, etc...

I want to assign `;` to repeat those motions. What would I have to write, exactly? I can't see how to write the code that does that, even after reading the readme. I am thinking something along the lines of: 

```elisp
(use-package repeatable-motion
  :ensure t
  :config
  (repeatable-motion-define 'every-single-evil-motion 'to-be-repeatable)
  ;; what function do I bind `;` to? 
)
```

Do you see my problem?

**Comment by willghatch** [comment]
Yeah, my documentation wasn't great.  I've updated the readme to have some more examples that will hopefully clarify things.  Let me know if it makes sense now.  Any other suggestions to make it more understandable would be welcome as well.

**Comment by ninrod** [comment]
thanks @willghatch, will try the new instructions and write feedback.
